### ISSUE TITLE
Quilt Loader doesn't show fabric mods

### ISSUE THREAD

**Issue by GhostIsBeHere** [ATLauncher/ATLauncher#575]
**Describe the bug**
When using the "Add Mods" button, Modrinth will filter by Quilt mods only, not fabric mods.

**To Reproduce**

1. Create a new Quilt instance
2. Click "Add Mods" button
3. Switch to Modrinth
4. Search for a fabric-only mod (I used sodium)

**Expected behavior**
Be able to view and download fabric-only mods, as quilt supports them

**System information (please complete the following information, or remove if not applicable):**

- MacOSX Mojave (10.14.6)
- Java 17

**Comment by RyanTheAllmighty** [comment]
Fabric mods that are compatable with Quilt should be tagged with both loaders, same way dual loaders are done for Forge/Fabric. Some mod devs have already done so, others not.

**Comment by GhostIsBeHere** [comment]
As all fabric mods are currently compatible with quilt, I still feel this change is important

**Comment by RyanTheAllmighty** [comment]
Yeah looks like another situation similar to what happened with CurseForge where dual loader mods didn't end up getting tagged for the longest time. I'll add this in for now, but will reassess later on if it should work that way or not as mod developers should be tagging it, but I realise not all will
